index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header >
+        <div class="logo">
+            <a href="">
+            <h3>Blessing Oluchi</h3>
+            </a>
+            <nav class="Hamburger">
+                <img  src="images/Hamburger2.png" alt="Hamburger">
+            </nav>
+        </div>
+        <section class="header-Bg">
+            <div class="header-Page">
+                <h1>I'm Blessing, <br> Glad to see <br> you!</h1>
+                <p>I'm a software developer! I can help you <br>
+                    build a product , feature or website Look <br>
+                    through some of my work and <br>
+                    experience! If you like what you see and <br>
+                    have a project you need coded, don't <br>
+                    hestiate to contact me.
+                </p>
+                <h6>LET'S CONNECT</h6>
+                <ul>
+                    <li><a href=""><img src="images/twitter2.png" alt="twitter"></a></li>
+                    <li><a href=""><img src="images/linkedIn2.png" alt="linkedIn"></a></li>
+                    <li><a href=""><img src="images/medium2.png" alt="medium"></a></li>
+                    <li><a href=""><img src="images/github2.png" alt="github"></a></li>
+                    <li><a href=""><img src="images/angelList2.png" alt="angelList"></a></li>
+                </ul>
+            </div>
+        </section>
```

Comment:
- [Optional] Dear @Luchy-B, Good job making the changes as requested, I see you made your social media icons `display: flex`, and it's good. But as a main requirement, we need to make the `Headline section` to  `display: flex`.